pmm/pmm2-upgrade-portal-tests.groovy

```diff
@@ -0,0 +1,288 @@
+            name: 'GIT_BRANCH')
+        string(
+            defaultValue: '',
+            description: 'Commit hash for the branch',
+            name: 'GIT_COMMIT_HASH')
+        choice(
+            choices: versionsList + ['custom'],
+            description: 'Docker tag for PMM Server Version',
+            name: 'PMM_SERVER_VERSION')
+        string(
+            defaultValue: '',
+            description: 'Custom version of PMM Server',
+            name: 'PMM_SERVER_VERSION_CUSTOM')
+        choice(
+            choices: ['perconalab/pmm-server', 'percona/pmm-server'],
+            description: "Docker hub for PMM Server",
+            name: 'PMM_DOCKER_HUB')
+        choice(
+            choices: versionsList + ['custom'],
+            description: 'PMM Client Version',
+            name: 'PMM_CLIENT_VERSION')
+        string(
+            defaultValue: '',
+            description: 'Custom version of PMM Client',
+            name: 'PMM_CLIENT_VERSION_CUSTOM')
+        choice(
+            choices: ['Experimental', 'Testing'],
+            description: "Select Testing (RC Tesing) or Experimental (dev-latest testing) Repository",
+            name: 'PMM_REPOSITORY')
+        string(
+            defaultValue: 'admin-password',
+            description: 'Change pmm-server admin user default password.',
+            name: 'ADMIN_PASSWORD')
+        string(
+            defaultValue: 'main',
+            description: 'Tag/Branch for pmm-qa repository',
+            name: 'PMM_QA_GIT_BRANCH')
+    }
+    options {
+        skipDefaultCheckout()
+    }
+    stages {
+        stage('Prepare') {
+            steps {
+                // clean up workspace and fetch pmm-ui-tests repository
+                deleteDir()
+                git poll: false, branch: GIT_BRANCH, url: 'https://github.com/percona/pmm-ui-tests.git'
+                wrap([$class: 'BuildUser']) {
+                    sh """
+                        echo "\${BUILD_USER_EMAIL}" > OWNER_EMAIL
+                        echo "\${BUILD_USER_EMAIL}" | awk -F '@' '{print \$1}' > OWNER_FULL
+                    """
+                }
+                script {
+                    slackSend botUser: true, channel: '#pmm-ci', color: '#FFFF00', message: "[${JOB_NAME}]: build started - ${BUILD_URL}"
+                }
+                sh '''
+                    sudo yum -y update --security
+                    sudo yum -y install php php-mysqlnd php-pdo jq svn bats mysql
+                    sudo amazon-linux-extras install epel -y
+                    sudo mkdir -p /srv/pmm-qa || :
+                    pushd /srv/pmm-qa
+                        sudo git clone --single-branch --branch \${PMM_QA_GIT_BRANCH} https://github.com/percona/pmm-qa.git .
+                        sudo git checkout \${PMM_QA_GIT_COMMIT_HASH}
+                        sudo chmod 755 pmm-tests/install-google-chrome.sh
+                        bash ./pmm-tests/install-google-chrome.sh
```

Comment:
We already have chromium on the agent:
https://github.com/percona/pmm-infra/blob/main/packer/ansible/agent.yml#L74
Or do we need chrome?